framework/source/class/qx/ui/tabview/TabView.js

```diff
@@ -563,7 +563,6 @@ qx.Class.define("qx.ui.tabview.TabView",
       {
         value = [button.getUserData("page")];
         pane.setSelection(value);
-        button.focus();
```

Comment:
It's easier to set focus manually (or add a method parameter for that) than avoid this behavior (I had to override the method).